setup.py

```diff
@@ -113,4 +113,21 @@
     license='Apache License, Version 2.0',
     maintainer="Phillip Cloud",
     maintainer_email="phillip.cloud@twosigma.com",
+    entry_points={
+        'ibis.backends': [
+            'pandas = ibis.pandas.api'
+            'csv = ibis.file.csv',
```

Comment:
i think if you sync this now ,then the updates for the other backends can update this as needed